game/view/hand.lua

```diff
@@ -119,12 +112,33 @@ function HandView:positionForIndex(i)
          y - 50 + (0.2+enter*0.4)*(i - (size+1)/2)^2*_GAP
 end
 
+function HandView:hide()
+  self.hiding = true
+end
+
+function HandView:show()
+  self.hiding = false
+end
+
 function HandView:update(dt)
-  self:reset()
+  local _FADE_SPD = 2
```

Comment:
why is this variable declared like a file constant?

Reply:
I think we're going to move it up but forgot. On it.